packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -55,17 +57,25 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     }
 
     public transformSchema(metadata: GraphbackCoreMetadata): GraphQLSchema {
-        const schema = metadata.getSchema()
+        let schema = metadata.getSchema();
         const models = metadata.getModelDefinitions();
         if (models.length === 0) {
             this.logWarning("Provided schema has no models. Returning original schema without any changes.")
 
             return schema;
-        }
+        };
+
+        schema = this.buildSchemaModelRelationships(schema, models);
+
+        // Print and rebuild to keep field and type descriptions
+        // WORKAROUND!
+        schema = buildSchema(printSchema(schema));
 
         const modelsSchema = this.buildSchemaForModels(models);
 
-        return mergeSchemas({ schemas: [modelsSchema, schema] });
+        schema = mergeSchemas({ schemas: [modelsSchema, schema] });
```

Comment:
Generally this is very convoluted API - we do merge schemas while using graphql compose that should deal with it